libnavigation-ui/src/main/java/com/mapbox/navigation/ui/voice/MapboxSpeechPlayer.java

```diff
@@ -127,7 +127,7 @@ private void stopMediaPlayerPlaying() {
   }
 
   private void downloadVoiceFile(final String instruction, String textType) {
-    boolean isInvalidInstruction = TextUtils.isEmpty(instruction);
+    boolean isInvalidInstruction = TextUtils.isEmpty(instruction) || instruction == null;
```

Comment:
```suggestion
    boolean isInvalidInstruction = TextUtils.isEmpty(instruction);
```

`TextUtils.isEmpty` already checks that 👀 

```
/**
* Returns true if the string is null or 0-length.
* @param str the string to be examined
* @return true if str is null or zero length
*/
public static boolean isEmpty(@Nullable CharSequence str) {
        return str == null || str.length() == 0;
}
```

Reply:
Good point 👍 Thanks